configs:
  mode:
    runner: "usb"
    temporalSubtraction: false
  sensor:
    general:
      fpsBoost: true
      brightness:
        alpha: 1.0
        beta: 0.0
    usbCam:
      maskSize: 0.8
      flipImage: true
      enableMask: true
      ports:
        lCam: "/dev/video4"
        rCam: "/dev/video6"
  algorithm:
    process:
      alignment:
        matchRate: 0.4
        usePreset: true
        maxFeatures: 500
      channel: "g" # ["b", "g", "r", "all"]
      subtractRL: true
      colorRange:
        hsv_green:
          lower: [35, 120, 50] # [35, 120, 50]
          upper: [85, 255, 255] # [85, 255, 255]
    postprocess:
      threshold:
        size: 50
        method: "binary" # ["binary", "otsu", "both"]
      invertBinary: false
      erosionKernel: 1
      gaussianKernel: 1
  marker:
    structure:
      size: 0.1
    detection:
      dictionary: "DICT_ARUCO_ORIGINAL"
